lib/neo4j-core/query_find_in_batches.rb

```diff
@@ -18,7 +18,8 @@ def find_in_batches(node_var, prop_var, options = {})
 
           break if records_size < batch_size
 
-          records = query.where("#{node_var}.#{prop_var} > {primary_key_offset}").params(primary_key_offset: primary_key_offset).to_a
+          records = query.where("#{Neo4j::Core::QueryClauses::Clause.from_key_and_single_value(node_var, prop_var)} > {primary_key_offset}")
```

Comment:
I would pull out the call to `from_key_and_single_value` to store a variable so that this line is a bit easier to parse